### DISCUSSION TITLE
Show slide pagination

### DISCUSSION THREAD

**Discussion by iamdarkle** [fancyapps/ui#549] [General]
Hi, I am thinking of migrating from Swiper + PhotoSwipe to the recent version of Fancybox which seems to include everything I need, I have come across a small detail that I am not sure if Fancybox provides the option, currently with swiper pagination in the top corner it shows the number of slides, is this possible in Fancybox?

![Screenshot 2023-08-08 150009](https://github.com/fancyapps/ui/assets/15818451/a5e6c81f-6282-4cbe-850c-143577b03ac3)

**Comment by fancyapps** [comment]
Hi, Do you mean this here?

![hmm](https://github.com/fancyapps/ui/assets/1055589/130635f5-adb2-4d4c-959c-5023eb62311b)

**Reply by iamdarkle** [reply]
Yes, basically that but outside, in the carousel before opening the fancybox.

**Reply by fancyapps** [reply]
You can use events to update the value of an element like in this demo - https://stackblitz.com/edit/js-jgb9qw?file=index.html,index.js,style.css

**Reply by iamdarkle** [reply]
Oh I see! thanks for the example, closing as resolved.
